lessons/introR/main.Rmd

```diff
@@ -0,0 +1,291 @@
+---
```

Comment:
So the only thing that may disrupt Jekyll is if I put a 'layout: post' tag in the YAML header. Otherwise, the site can link directly to the html files I created.